doc/simd_api.rst

```diff
@@ -0,0 +1,948 @@
+
+    * - ``S::simd_mask``
+      - ``simd_mask<V, N, I>``
+      - The ``simd_mask`` specialization resulting from comparisons of *S* SIMD values.
+
+    * - ``S::width``
+      - ``unsigned``
+      - The SIMD width *N*.
+
+.. rubric:: Constructors
+
+.. list-table::
+    :widths: 20 80
+    :header-rows: 1
+
+    * - Expression
+      - Description
+
+    * - ``S(x)``
+      - A SIMD value *v* with *v*\ `i`:sub: equal to *x* for *i* = 0…*N*-1.
+
+    * - ``S(t)``
+      - A copy of the SIMD value *t*.
+
+    * - ``S(c)``
+      - A SIMD value *v* with *v*\ `i`:sub: equal to ``c[i]`` for *i* = 0…*N*-1.
+
+    * - ``S(c, m)``
+      - A SIMD value *v* with *v*\ `i`:sub: equal to ``c[i]`` for *i* where *m*\ `i`:sub: is true.
+
+.. rubric:: Member functions
+
+.. list-table::
+    :widths: 20 20 60
+    :header-rows: 1
+
+    * - Expression
+      - Type
+      - Description
+
+    * - ``t.copy_to(p)``
+      - ``void``
+      - Set ``p[i]`` to *t*\ `i`:sub: for *i* = 0…*N*-1.
+
+    * - ``t.scatter(p, j)``
+      - ``void``
+      - Set ``p[j[i]]`` to *t*\ `i`:sub: for *i* = 0…*N*-1.
+
+    * - ``s.copy_from(c)``
+      - ``void``
+      - Set *s*\ `i`:sub: to ``c[i]`` for *i* = 0…*N*-1.
+
+    * - ``s.gather(c, j)``
+      - ``void``
+      - Set *s*\ `i`:sub: to ``c[j[i]]`` for *i* = 0…*N*-1.
+
+.. rubric:: Expressions
+
+.. list-table::
+    :widths: 20 20 60
+    :header-rows: 1
+
+    * - Expression
+      - Type
+      - Description
+
+    * - ``t+u``
+      - ``S``
+      - Lane-wise sum.
+
+    * - ``t-u``
+      - ``S``
+      - Lane-wise difference.
+
+    * - ``t*u``
+      - ``S``
+      - Lane-wise product.
+
+    * - ``t/u``
+      - ``S``
+      - Lane-wise quotient.
+
+    * - ``fma(t, u, v)``
+      - ``S``
+      - Lane-wise FMA *t* * *u* + *v*.
+
+    * - ``s<t``
+      - ``S::simd_mask``
+      - Lane-wise less-than comparison.
+
+    * - ``s<=t``
+      - ``S::simd_mask``
+      - Lane-wise less-than-or-equals comparison.
+
+    * - ``s>t``
+      - ``S::simd_mask``
+      - Lane-wise greater-than comparison.
+
+    * - ``s>=t``
+      - ``S::simd_mask``
+      - Lane-wise greater-than-or-equals comparison.
+
+    * - ``s==t``
+      - ``S::simd_mask``
+      - Lane-wise equality test.
+
+    * - ``s!=t``
+      - ``S::simd_mask``
+      - Lane-wise inequality test.
+
+    * - ``s=t``
+      - ``S&``
+      - Lane-wise assignment.
+
+    * - ``s+=t``
+      - ``S&``
+      - Equivalent to ``s=s+t``.
+
+    * - ``s-=t``
+      - ``S&``
+      - Equivalent to ``s=s-t``.
+
+    * - ``s*=t``
+      - ``S&``
+      - Equivalent to ``s=s*t``.
+
+    * - ``s/=t``
+      - ``S&``
+      - Equivalent to ``s=s/t``.
+
+    * - ``s=x``
+      - ``S&``
+      - Equivalent to ``s=S(x)``.
+
+    * - ``t[i]``
+      - ``V``
+      - Value *t*\ `i`:sub:
+
+    * - ``s[i]=x``
+      - ``S::reference``
+      - Set value *s*\ `i`:sub: to *x*.
+
+The (non-const) index operator `operator[]` returns a proxy object of type `S::reference`,
```

Comment:
the `operator[]` and other code quotes need double quoting for rst.

Reply:
Fixed.